client/gutenberg/extensions/tiled-gallery/index.js

```diff
@@ -0,0 +1,186 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+import { filter } from 'lodash';
+import { Rect, SVG } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { __, _x } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { default as edit } from './edit';
```

Comment:
I know this was like that before, but couldn't we just have `import edit from './edit'` here?

Reply:
yeah, that would read better